spec/models/category_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  describe "attributes" do
+    let!(:category) { Category.create!(name: "Technology") }
```

Comment:
`let!` will *always* create `category` for every test case. Do you need it in every example below?